docs/external-template-creation.md

```diff
@@ -84,3 +72,42 @@ export default {
 ```
 
 Once you've completed this step, your new sticker should now appear with the other stickers appended to the bottom of the shapes panel if the `enableStickers` feature flag is enabled.
+
+### Inserting stickers (SVGs) into a story
+
+To add stickers to your template, it's as easy as going to the experiments and enabling the `enableStickers` flag. Once turned on, all available stickers in the codebase should be available for selection under our base shapes in the shapes library panel.
+
+### Get the story JSON
+
+To get the raw story json, simply open up a story in the web story editor with the devtools network tab open and press `save`. You should see the the story's json show up in the network tab. In that json, right click the `story_data` field and click `Store as global variable`. Go to the devtools console and type 
+
+```javascript
+copy(temp1)
+```
+
+When you press enter the story data should now be copied to your clipboard and ready to paste where you need.
+
+**Note**: `Command+Shift+Option+J / Control+Shift+Alt+J` in the story editor opens up a dialog you can copy/paste raw story json to/from. However this is missing some fields `story_data` contains which are necessary for versioning the external templates.
+
+### Adding story JSON to the codebase as a new template
+
+Once you've added your template's raw story json to `packages/templates/src/raw/<template_name>.json`, you can import it into `packages/templates/src/getTemplates` and add to the function `loadTemplates(imageBaseUrl)`. It should then be accessible in `packages/templates/src/index`.
+
+## Appendix
+
+### Filenames for images and videos
+
+WordPress has a naming convention where if you upload an image with the same name multiple times, it adds a `-x` suffix. 
+
+**i.e**
+if I upload `some_image.png` 3 times to the WordPress media upload in the story editor, it will store those images as:
+
+```javascript
+// some_image.png
+// some_image-1.png
+// some_image-2.png
+```
+
+Because of this, our conversion script will strip the `-x` suffix off of the image source so it can point to the correct image stored in our repo.
+
+This is an important note to make when naming images for upload into a story as **a `-x` suffix will always be stripped**. This naming convention is reserved for WordPress and will break your template creation if you use it in your image naming.
```

Comment:
@littlemilkstudio Are we still doing this? I never really understood why we're doing this at runtime. We should prevent this at the ESLint stage.

Reply:
It's been a while since I needed to update any media on a template, but yea I believe it's still going on.

We can add a lint step here, but honestly kinda hard to parse through the ~5000 line json files. We could try out testing this in the lint step tho and just see how it fares. Agree that it'd be nicer if it works well

Reply:
Can you point me to the code where this is being done?

Not worried about the size of the template files for these checks. We already have Jest tests to "lint" the templates. Adding one more for file names will be trivial.

Reply:
> Can you point me to the code where this is being done?

Sorry, not completely following. Where what's being done? the appending of the `-n` on image file names if uploaded multiple times?

Reply:
"our conversion script will strip the `-x` suffix off"

Where is this happening? I remember seeing this, but can't find it right now.

Reply:
🤔 I can't seem to find it either. I'm guessing we got rid of it since the last time we were in here 🤷 . Feel like if it'd be anywhere, would be in the `loadTemplate` function